wavefront_python_sdk/common/connection_handler.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import, division, print_function
+
+from wavefront_python_sdk.common.utils import AtomicCounter
+
+
+class ConnectionHandler:
```

Comment:
If you want your code to be python 2.x compatible, you may want to put `object` as your parent class explicitly here and elsewhere, e.g. `class ConnectionHandler(object):`